clients/admin-ui/src/features/system/form.ts

```diff
@@ -102,6 +102,11 @@ export const transformFormValuesToSystem = (formValues: FormValues): System => {
     vendor_id: formValues.vendor_id,
     ingress: formValues.ingress,
     egress: formValues.egress,
+    meta: formValues.meta,
+    fidesctl_meta: formValues.fidesctl_meta,
+    registry_id: formValues.registry_id,
```

Comment:
We have a weird thing with the registry_id looks like. I have no idea what this field is for, but validation expects an integer, but database needs it to be a string.  If FE is sending an integer here, that will fail at the db level.  Maybe we leave out this one.

Reply:
registries are an extremely old and never-used feature...honestly should just remove it

It was conceived as a way to help organizations organize their fides resources into sub-units. Specifically you could assign systems to a registry to logically "group" them. This use-case has never materialized

Reply:
ah okay thanks for the background Thomas!